hubblestack/extmods/modules/audit.py

```diff
@@ -1,450 +1,726 @@
-            ret['Skipped'].append({audit_id: data})
-            continue
-        elif version:
-            version_match = _version_cmp(version)
-            if not version_match:
-                log.debug(
-                    'Skipping audit %s due to version %s not matching version requirements %s',
-                    audit_id, __grains__['hubble_version'], version)
-                ret['Skipped'].append({audit_id: data})
-                continue
 
-        args = data.get('args', [])
-        kwargs = data.get('kwargs', {})
+def _get_audit_files(audit_files):
+    """Get audit files list, if valid
 
-        # Run the audit
-        try:
-            success, data_dict = __audit__[module](*args, **kwargs)
-        except Exception as exc:
-            log.error('Audit %s from file %s failed with exception %s',
-                      audit_id, audit_file, exc)
-            data['reason'] = 'exception'
-            data['exception'] = str(exc)
-            ret['Failure'].append({audit_id: data})
-            continue
+    Arguments:
+        audit_files {str or list} -- File lists either in comma-separated or python-list
 
-        if data_dict and isinstance(data_dict, dict):
-            data_dict.update(data)
-        else:
-            data_dict = data
+    Returns:
+        list -- List of audit files
+    """
+    if not audit_files:
+        log.warning('nova.audit called without any audit_files')
+        return None
 
-        if success:
-            ret['Success'].append({audit_id: data_dict})
-        else:
-            ret['Failure'].append({audit_id: data_dict})
+    if not isinstance(audit_files, list):
+        audit_files = audit_files.split(',')
 
-    return ret
+    # prepare paths
+    return ['salt://' + BASE_DIR_NOVA_PROFILES + os.sep + audit_file.replace('.', os.sep) + '.yaml'
+            for audit_file in audit_files]
 
 
-def _process_data(data, tags, labels, label_list):
-    """ Validate data, log if necessary, return False to skip the audit, True to continue """
-    audit_id = data['audit_id']
-    tag = data.get('tag', audit_id)
+def _evaluate_results(result_dict, combined_dict, show_compliance, verbose):
+    """
+    Evaluate the result dictionary to be returned by the audit module
+    :param result_dict: Final dictionary to be returned
+    :param combined_dict: Initial dictionary with results for all profiles
+    :param show_compliance: Para to show compliance percentage
+    :param verbose: Create output in verbose manner or not
+    :return:
+    """
+    for audit_file in combined_dict:
+        result_list = combined_dict[audit_file]
+        for result in result_list:
+            sub_check = result.get('sub_check', False)
+            if not sub_check:
+                dict = {}
+                if verbose:
+                    dict[result['tag']] = result
+                else:
+                    dict[result['tag']] = result['description']
```

Comment:
In non-verbose mode, we should probably use either the tag (`ADOBE-00033`) or the check_id.